portal/src/util/deno.ts

```diff
@@ -1,3 +1,7 @@
-const authgear_deno_hook_version = "v1.5.0";
+// FIXME(tung): Use the published version
+// const authgear_deno_hook_version = "v1.5.0";
```

Comment:
Use v1.6.0

Please also update the non-blocking hook template to use `HookNonBlockingEvent`

Reply:
In case you missed this comment.